reportssql/indicator_functions.sql

```diff
@@ -913,6 +913,39 @@ WHERE
 END$$ 
 DELIMITER ;
 
+DROP FUNCTION IF EXISTS Testing_Indicator3a;
+
+DELIMITER $$
+CREATE FUNCTION Testing_Indicator3a(
+    p_startDate DATE,
+    p_endDate DATE,
+    p_startAgeInMonths INT(11),
+    p_endAgeInMonths INT (11),
+    p_includeStartAge TINYINT(1)) RETURNS INT(11)
+    DETERMINISTIC
+BEGIN
+    DECLARE result INT(11) DEFAULT 0;
+
```

Comment:
@tmvumbi2 We can define dateOfVirologicHIVTest here:

DECLARE dateOfVirologicHIVTest DATE;
SET dateOfVirologicHIVTest = getDateOfVirologicTest(p_patientId, p_startDate, p_endDate);